### PR TITLE
my grid layout

### PR DESCRIPTION
**Pull request by Paulooodev**

### CODE REVIEW

File: .vscode/index2.css
Comment (Sesughter01) [inline .vscode/index2.css:1]: Please remove all the files you have created in the  .vscode folder  you . The .vscode folder holders debugger settings you not to create any files or folders inside it. Remove all the files it currently except what was there before which is the settings.json. Remove the index2.css and index2.html

File: .vscode/index2.html
Comment (Sesughter01) [inline .vscode/index2.html:1]: Please remove all the files you have created in the .vscode folder you . The .vscode folder holders debugger settings you not to create any files or folders inside it. Remove all the files it currently except what was there before which is the settings.json. Remove the index2.css and index2.html

Comment (Paulooodev) [reply .vscode/index2.html:1]: Okay sir

File: class_six/index.html
Comment (Sesughter01) [inline class_six/index.html:1]: Revert back to initial head tag content to help make your site more responsive and adopt best practices

**Review by Sesughter01 — COMMENTED** [review]
Make changes accordingly

File: class_six/index.html
Comment (Paulooodev) [reply class_six/index.html:1]: Okay sir

**Pull request closed without merging**